recipes/wxwidgets/all/conanfile.py

```diff
@@ -0,0 +1,477 @@
+    homepage = "https://www.wxwidgets.org"
+    license = "wxWidgets"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "jpeg": ["libjpeg", "libjpeg-turbo", "mozjpeg"],
+               "secretstore": [True, False],
+               "aui": [True, False],
+               "opengl": [True, False],
+               "html": [True, False],
+               "mediactrl": [True, False],
+               "propgrid": [True, False],
+               "debugreport": [True, False],
+               "ribbon": [True, False],
+               "richtext": [True, False],
+               "sockets": [True, False],
+               "stc": [True, False],
+               "webview": [True, False],
+               "xml": [True, False],
+               "xrc": [True, False],
+               "cairo": [True, False],
+               "help": [True, False],
+               "html_help": [True, False],
+               "url": [True, False],
+               "protocol": [True, False],
+               "fs_inet": [True, False],
+               "custom_enables": ["ANY"], # comma splitted list
+               "custom_disables": ["ANY"]}
+    default_options = {
+               "shared": False,
+               "fPIC": True,
+               "jpeg": "libjpeg",
+               "secretstore": True,
+               "aui": True,
+               "opengl": True,
+               "html": True,
+               "mediactrl": False,
+               "propgrid": True,
+               "debugreport": True,
+               "ribbon": True,
+               "richtext": True,
+               "sockets": True,
+               "stc": True,
+               # WebKitGTK for GTK2 is not available as a system dependency on modern distros.
+               # When gtk/system defaults to GTK3, turn this back on.
+               "webview": False,
+               "xml": True,
+               "xrc": True,
+               "cairo": True,
+               "help": True,
+               "html_help": True,
+               "url": True,
+               "protocol": True,
+               "fs_inet": True,
+               "custom_enables": "",
+               "custom_disables": ""
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+        if self.settings.os != "Linux":
+            self.options.rm_safe("secretstore")
+            self.options.rm_safe("cairo")
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    @property
+    def _gtk_version(self):
+        return f"gtk{self.dependencies['gtk'].options.version}"
+
+    def system_requirements(self):
+        apt = package_manager.Apt(self)
+        packages = []
+        if self.options.get_safe("secretstore"):
+            packages.append("libsecret-1-dev")
```

Comment:
There is libsecret in conancenter.

I'm surprised by this recipe. I thought it was forbidden to have system requirements explicitly requested in conancenter (except for a "system" recipe).

Reply:
Same question. How did this get past hook checks and human reviews?